### PR TITLE
Update dependencies version and add Jenkins pipeline

### PR DESCRIPTION
**Pull request by josepvilabadillo**
Updated dependencies version:
- Added option for boost 1.76
- Upgraded TestUtilities to 1.0.8

Fixed adapter interface pipeline to not include settings in package.
Added jenkins files to build the library.

### CODE REVIEW

File: DbAdapterInterface/conanfile.py [hunk 1]
```diff
@@ -10,8 +10,7 @@ class DbAdapterInterfaceConan(ConanFile):
     topics = ("conan", "db", "sql", "adapter", "wrapper")
     license = "MIT"
     generators = "cmake_find_package"
-    settings = "os", "compiler", "build_type", "arch"
-    options = {"boost": ["1.66.0", "1.67.0", "1.72.0", "1.75.0"]}
+    options = {"boost": ["1.66.0", "1.67.0", "1.72.0", "1.75.0", "1.76.0"]}
     default_options = {"boost": "1.72.0"}
```
Comment (PacoGutierrez) [inline DbAdapterInterface/conanfile.py]: Why boost version by default is 1.72.0 instead lastest version?

File: DbAdapterTestUtilities/conanfile.py [hunk 2]
```diff
@@ -12,7 +12,7 @@ class DbAdapterTestUtilitiesConan(ConanFile):
     license = "MIT"
     generators = "cmake_find_package"
     settings = "os", "compiler", "build_type", "arch"
-    options = {"boost": ["1.66.0", "1.67.0", "1.72.0", "1.75.0"], "gtest": ["1.7.0", "1.8.1", "1.10.0"]}
+    options = {"boost": ["1.66.0", "1.67.0", "1.72.0", "1.75.0", "1.76.0"], "gtest": ["1.7.0", "1.8.1", "1.10.0"]}
     default_options = {"boost":"1.72.0", "gtest": "1.10.0"}
```
Comment (PacoGutierrez) [inline DbAdapterTestUtilities/conanfile.py]: The same here. However we are using latest gtest version.

**Review by PacoGutierrez — CHANGES_REQUESTED** [review]

File: DbAdapterInterface/conanfile.py (hunk 1 above)
Comment (josepvilabadillo) [reply DbAdapterInterface/conanfile.py]: Yes, good point, we should be using the latest version if not provided. Fixed.

File: DbAdapterTestUtilities/conanfile.py (hunk 2 above)
Comment (josepvilabadillo) [reply DbAdapterTestUtilities/conanfile.py]: Yes, good point, we should be using the latest version if not provided. Fixed.

**Review by PacoGutierrez — APPROVED** [review]

**Review by PacoGutierrez — COMMENTED** [review]
Requested changes have been made

**Pull request merged**